### ISSUE TITLE
Task 0

### ISSUE THREAD

**Issue by samuel0007** [samuel0007/introML#1]
TASK
This task is a trivial form of regression: Your goal is to predict a value y based on a vector x. While the exact relationship is usually not known, in this task, y is the mean of x. You may verify this on the provided training set. Your task is to make predictions for y on the provided test set.
